src/doc/trpl/iterators.md

```diff
@@ -242,6 +242,9 @@ for num in nums.iter() {
 }
 ```
 
+Sometimes you need this functionality, but since for loops operate on the
+`IntoIterator` trait, calling `.iter()` is rarely necessary.
```

Comment:
Can you suggest a better wording? I'm afraid I don't know very much about iterator adaptor methods.